std/uri.d

```diff
@@ -287,6 +294,13 @@ if (isSomeChar!Char)
     return r;
 }
 
+///
+@safe unittest
+{
+    assert("foo&/".encode == "foo&/");
```

Comment:
This should be an example for `decode`

Reply:
There was a small typo before but this unittest is for `encode`